packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -297,7 +297,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
       const axisDisplayOptions = {
         border: {
           show: customConfig.axisBorderShow || false,
-          width: 1,
+          width: 1 / devicePixelRatio,
```

Comment:
unrelated drive-by fix. this maintains 1 device pixel width instead of 1 css pixel width at different pixel density displays.

it's uniform with how the grid renders, but it's actually kind of odd that we do this. the grid stroke width should scale the same as the series stroke widths and axis labels strokes. but changing this to 1 css pixel causes the grid to become more prominent, so will need more consensus.